apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -834,7 +834,9 @@ export class LemonadePipeline implements BasePipeline {
 
       const ticketActions: PCDAction[] = [];
 
-      if (this.loaded) {
+      // If the Atom DB is not empty, i.e. if we have loaded atoms for this pipeline
+      // at least once since the server started, delete the folder.
+      if ((await this.db.load(this.id)).length > 0) {
```

Comment:
This has a cost (to fetch from the DB) so I have some thoughts on reducing duplication of that work:
1) It seems like there's duplicate effort from loading the tickets above.  Is it difficult to get this information from the same fetch.  Maybe it's tricky since there could be no tickets for the user's email.  At least if tickets is non-empty we can assume there are atoms without fetching again, right?
2) It should at least be possible to do this only once per pipeline instance if the first time you do this fetch you set `this.everLoaded`.

There are some assumptions here, like the fact that the atom DB is never empty in an active/loaded system.  What would happen if an admin simply deleted all the tickets?  Would that mean no atoms, and no delete, so that the client-side cleanup would never happen?  Or are atoms a more persistent concept than that?